package.json

```diff
@@ -128,5 +129,6 @@
     "expo": {
       "optional": true
     }
-  }
```

Comment:
unrelated line:
I see many errors on CI in regard to E2E and Samples tests, is it ready for review?

Reply:
I saw your mention on another PR that it will be fixed on the main PR so I will resolve this conversation